README.md

```diff
@@ -64,6 +64,7 @@ There also exists the PyroScan object which allows you to make a N-D parameter s
 ## Supports sources of Equilibrium data
 pyrokinetics currently supports
 * GEQDSK
+* TRANSP
```

Comment:
If you have documentation that defines the data format is it possible to link to it/give a reference? Helpful to see where later implementation comes from etc.

Reply:
No clear documentation, this was an educated guess that turned out to be correct. This is the closest thing I could find 
https://w3.pppl.gov/~pshare/help/body_transp_hlp.html#outfile56.html

Reply:
Thanks, in that case it might be worth including something in the docs about what you're assuming/guessing so that there's a record etc. I did have the GEQDSK documentation somewhere and I'd guess you do as well. Handy to link to as these things can be hard to find.